taichi/transforms/ir_printer.cpp

```diff
@@ -62,7 +62,7 @@ class IRPrinter : public IRVisitor {
 
   void print_raw(std::string f) {
     for (int i = 0; i < current_indent; i++)
-      f = "  " + f;
+      f.insert(0, "  ");
```

Comment:
is this change relevant?

Reply:
This is the only warning in our codebase detected by current rules. It is used to test whether CI result changes as expected.